RELEASE_NOTES.md

```diff
@@ -1,5 +1,32 @@
 # Release Notes
 
+# Staging 2019/05/22 
+## Ingest
+### Exporter v0.8.1.rc
+* Do not inform user when there’s a failure creating a duplicate bundle
+
```

Comment:
@ranthony sorry, I just added some more release notes for Ingest UI that was also deployed to staging today. Could you add it here?

Reply:
Sure. Np :)

Reply:
Done!

Reply:
Thank you! :)